### DISCUSSION TITLE
Since Spotube relies on API's from both YouTube and Spotify, how likely is it to continue working in the long run?

### DISCUSSION THREAD

**Discussion by Amplifiction** [KRTirtho/spotube#2027] [Q&A]
See title. I imagine both companies would like us to use their original apps. How hard is it for them to change their API's so that Spotube ceases to function? I'm not rooting for that to happen, to be clear. On the contrary, hence my concern.

**Comment by KRTirtho** [comment]
We don't know. As long as it works.
